FAIL
Kernel panic in run: PANIC at ../../threads/interrupt.c:361 in intr_handler(): Unexpected interrupt
Call stack: 0x80042143cd 0x8004208d8e 0x8004209171 0x800421461c 0x8004215249 0x800421566d 0x8004209f52 0x800420a5f6 0x8004218eb8 0x8004207338
Translation of call stack:
0x00000080042143cd: debug_panic (lib/kernel/debug.c:32)
0x0000008004208d8e: intr_handler (threads/interrupt.c:365)
0x0000008004209171: intr_entry (intr-stubs.o:?)
0x000000800421461c: list_next (lib/kernel/list.c:78)
0x0000008004215249: find_end_of_run (lib/kernel/list.c:347 (discriminator 2))
0x000000800421566d: list_sort (lib/kernel/list.c:401)
0x0000008004209f52: sema_up (threads/synch.c:113)
0x000000800420a5f6: lock_release (threads/synch.c:287)
0x0000008004218eb8: acquire1_thread_func (tests/threads/priority-donate-one.c:53)
0x0000008004207338: kernel_thread (threads/thread.c:400)
